frontend/src/component/feature/FeatureForm/FeatureForm.tsx

```diff
@@ -128,16 +136,52 @@ const FeatureForm: React.FC<IFeatureToggleForm> = ({
         return featureTypes.find(toggle => toggle.id === type)?.description;
     };
 
+    const displayFeatureNamingInfo = Boolean(featureNaming?.pattern);
+
     return (
         <StyledForm onSubmit={handleSubmit}>
             <StyledContainer>
                 <StyledInputDescription>
                     What would you like to call your toggle?
                 </StyledInputDescription>
+                <ConditionallyRender
+                    condition={displayFeatureNamingInfo}
+                    show={
+                        <StyledFlagNamingInfo>
+                            <p>
+                                This project has feature flag naming patterns
+                                enabled.
+                            </p>
+                            <dl id="feature-naming-pattern-info">
+                                <dt>Pattern</dt>
+                                <dd>
+                                    <code>{featureNaming?.pattern}</code>
+                                </dd>
+                                {Boolean(featureNaming?.example) && (
```

Comment:
Could use conditional render component for readability

Reply:
Yeah, I thought about that, but decided against it because of the layers of nesting. Felt like it would take up a lot of extra vertical space for not a lot of extra value. However, I do see your point and would be happy to switch to the component instead if you think that's better ☺️